scripts/core/editor3/reducers/table.tsx

```diff
@@ -94,41 +128,47 @@ const removeRow = (state) =>
  * @description Adds a column after the currently active one.
  */
 const addColAfter = (state) =>
-    processCells(state, (cells, numCols, numRows, i, j, withHeader) => ({
-        data: {
-            numRows: numRows,
-            numCols: numCols + 1,
-            withHeader: withHeader,
-            cells: cells.map((_, ii) =>
-                Array.from(new Array(numCols + 1))
-                    .map((__, jj) => {
-                        if (jj === j + 1) {
-                            return null;
-                        }
+    processCells(state, (tableData, activeCell) => {
+        const {cells, numCols} = tableData;
+        const {j} = activeCell;
 
-                        let orig = jj;
+        return {
+            data: {
+                ...tableData,
+                numCols: tableData.numCols + 1,
+                cells: cells.map((_, ii) =>
```

Comment:
this `ii` looks a bit odd. Maybe do firstIndex and for `jj` do `secondIndex`

Reply:
I didn't write this code, just refactored arguments. Another thing is that inside active cell there's a convention that `i` is row and `j` is column. When working with matrices you want short index names so ii/jj is quite a good choice.